src/main/java/uk/gov/companieshouse/api/accounts/service/impl/DirectorService.java

```diff
@@ -231,4 +254,22 @@ private String getDirectorId(HttpServletRequest request) {
 
         return directorId;
     }
+
+    private void directorsResourceChanged(Transaction transaction, String companyAccountsId,
```

Comment:
All looks good, though I may recommend renaming this method to be more descriptive. Perhaps `removeAssociatedLoans` or similar?